ludwig/schema/metadata/configs/preprocessing.yaml

```diff
@@ -44,6 +44,13 @@ sample_ratio:
     expected_impact: 2
     suggested_values: Depends on data size
     ui_display_name: Sample Ratio
+column:
+    default_value_reasoning:
+        The default value is `split` because it is the column name that we traditionally reserve for indicating
+        dataset splitting.
+    expected_impact: 3
+    ui_display_name: Split Column
+    ui_component_type: column_selector
```

Comment:
are we adding this as a new component type?

Reply:
I'm adding in the column dropdown selector real quick.